frontend/simple/views/Invite.vue

```diff
@@ -98,22 +102,32 @@ export default {
       this.members.splice(index, 1)
     },
     submit: async function () {
-      for (let i = 0; i < this.members.length; i++) {
-        let member = this.members[i]
-        let events = await backend.eventsSince(member.contractId, member.contractId)
-        let [contract, ...actions] = events.map(e => {
-          return Events[e.entry.type].fromObject(e.entry, e.hash)
-        })
-        let state = contract.toVuexState()
-        actions.forEach(action => {
-          let type = action.constructor.name
-          contract.constructor.vuex.mutations[type](state, action.data)
-        })
-        let mailbox = await backend.latestHash(state.attributes.mailbox)
-        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId}, mailbox)
-        await backend.publishLogEntry(state.attributes.mailbox, invite)
+      try {
+        this.errorMsg = null
+        for (let i = 0; i < this.members.length; i++) {
+          let member = this.members[i]
+          let events = await backend.eventsSince(member.contractId, member.contractId)
+          let [contract, ...actions] = events.map(e => {
+            return Events[e.entry.type].fromObject(e.entry, e.hash)
+          })
+          let state = contract.toVuexState()
+          actions.forEach(action => {
+            let type = action.constructor.name
+            contract.constructor.vuex.mutations[type](state, action.data)
+          })
+          let mailbox = await backend.latestHash(state.attributes.mailbox)
```

Comment:
To someone who is completely unfamiliar with the code, this section will seem completely inscrutable and opaque as to what's going on.

So it could stand to benefit from comments explaining what's going on. I recommend:

- A top level comment on line 108 explaining that for each member it needs to get their latest mailbox attribute by building up the state from the identity contract log
- Then a TODO to put this "inline state building" paradigm in a function somewhere to respect DRY (since it's probably done in several locations)